packages/wallet/backend/src/user/model.ts

```diff
@@ -1,15 +1,15 @@
 import { Account } from '@/account/model'
 import { BaseModel } from '@/shared/model'
 import { Session } from '@/session/model'
-import { Model, QueryContext } from 'objection'
-import { randomBytes } from 'crypto'
-import { bcrypt, bcryptVerify } from 'hash-wasm'
+import { Model, ModelOptions, QueryContext } from 'objection'
+import { bcryptVerify } from 'hash-wasm'
+import { encryptPassword } from '@/utils/helpers'
 
 export class User extends BaseModel {
   static tableName = 'users'
 
   public email!: string
-  private password!: string
+  public password!: string
```

Comment:
I think we should find a way to keep the password `private` or exclude it from every response.